skglm/solvers/group_cd.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+from numpy.linalg import norm
+from numba import njit, jit
+
+from skglm.datafits.base import BaseMultitaskDatafit
+from skglm.penalties.base import BasePenalty
+
+from skglm.penalties.block_separable import SparseGroupL1
+from skglm.datafits.multi_task import QuadraticGroup
+
+from skglm.utils import make_correlated_data, grp_converter
+from celer import GroupLasso
+
+
+FREQ_CHECK = 10
```

Comment:
no need to expose them at submodule level, they can be defined inside `group_solver`